src/js/ripple/amount.js

```diff
@@ -9,6 +9,19 @@ var Seed      = require('./seed').Seed;
 var Currency  = require('./currency').Currency;
 var BigNumber = require('./bignumber');
 
+
+function isInteger(number) {
+  return parseInt(number) === number;
+}
+
+function noIntegers(value) {
```

Comment:
It feels kind of pointless commenting on this, given it's a transitional commit, but if this commit has any value, so does a comment, so here goes:

`noIntegers` ?

The number `1.0` IS an integer, as `isInteger` would tell you. 
You and I happen to know the purpose for this function, but other people I'm sure would benefit from a better name, and some explanatory comments.  Off the top of my head: `forceFloatRepr` or `forceDecimalPoint`